entityDumper/types.go

```diff
@@ -4,6 +4,19 @@ import (
 	"github.com/uyuni-project/inter-server-sync/utils"
 )
 
+type ExportConfig struct {
+	ServerConfig        string
+	Channels            []string
+	ChannelWithChildren []string
+	OutputDir           string
+	MetadataOnly        bool
+	StartingDate        string
+	ConfigChannels      []string
+	IncludeImages       bool
+	IncludeContainers   bool
+	Orgs                []uint
+}
```

Comment:
We can probably add an example of such config and then reflect it in the documentation like a quick start option or anything like it

Reply:
f497fb0